src/main/scala/fly/play/jiraExceptionProcessor/Jira.scala

```diff
@@ -46,14 +45,20 @@ class Jira(
   }
 
   private def findIssue(hash: String, componentId: String, hashCustomField: String) =
-    playProjectIssueFormat flatMap { implicit format =>
+    playProjectIssueFormat flatMap { implicit format => {
+      val searchOnHash = configuration.hashCustomFieldType match {
+        case JiraExceptionProcessorConfiguration.FieldType.UUID => s"${configuration.hashCustomFieldName} = $hash "
+        case JiraExceptionProcessorConfiguration.FieldType.TEXT => s"""${configuration.hashCustomFieldName} ~ \"\\\"$hash\\\"\" """
```

Comment:
Instead of using `s"""... ~ \"\\\"$hash\\\"\ """"` you could use `raw"""... ~ "\"$hash\"" """` to prevent the excessive escaping.